### PR TITLE
Filter statuses properly

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

**Comment by whalemer** [comment]
Also did you mean to bump the max status length from 18 to 32 chars?

File: server/chat.js [hunk 1]
```diff
@@ -272,6 +272,7 @@ Chat.nicknamefilters = [];
  * @param {boolean} [forStatus]
  */
 Chat.nicknamefilter = function (nickname, user, forStatus = false) {
+	if (forStatus) nickname = nickname.replace(/\|/g, '');
```
Comment (Zarel) [inline server/chat.js]: Nicknames shouldn't contain `\|` either.

**Comment by Zarel** [comment]
And yes, there's no reason to restrict statuses as severely as usernames.

File: server/chat.js (hunk 1 above)
Comment (Zarel) [reply server/chat.js]: (It's already enforced elsewhere, but there's no harm in also enforcing it here.)

**Comment by scheibo** [comment]
It doesn't fit on the usercard...

**Pull request merged**